### PR TITLE
coinex markets

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: js/coinex.js [hunk 1]
```diff
@@ -376,16 +376,11 @@ module.exports = class coinex extends Exchange {
...
-        const [ type, query ] = this.handleMarketTypeAndParams ('fetchMarkets', undefined, params);
-        if (type === 'spot' || type === 'margin') {
-            result = await this.fetchSpotMarkets (query);
-        } else if (type === 'swap') {
-            result = await this.fetchContractMarkets (query);
-        } else {
-            throw new ExchangeError (this.id + " does not support the '" + type + "' market type, set exchange.options['defaultType'] to 'spot', 'margin' or 'swap'");
-        }
-        return result;
+        let promises = [ this.fetchSpotMarkets (params), this.fetchContractMarkets (params) ];
```
Comment (kroitor) [inline js/coinex.js:379]: Shouldn't it be `const` here ?

Comment (kroitor) [inline js/coinex.js:379]: Let's make sure that they can be actually loaded all at once, cause it differentiated between them previously.

Comment (frosty00) [reply js/coinex.js:379]: ?

Comment (frosty00) [reply js/coinex.js:379]: the point is to load them all at once

Comment (kroitor) [reply js/coinex.js:379]: @frosty00 

> the point is to load them all at once

only when their ids don't conflict

Comment (frosty00) [reply js/coinex.js:379]: ah yeah i forget...

**Pull request closed without merging**